code/modules/mob/living/carbon/human/species/station/protean_vr/protean_blob.dm

```diff
@@ -746,8 +746,56 @@
 			I.plane = PLANE_LIGHTING_ABOVE
 			add_overlay(I)
 			qdel(I)
+		if(S.blob_appearance == "dullahan") //chomp edit
```

Comment:
^ // CHOMPEdit Start

Reply:
How does that look? Tried doing it from my phone